ses

```diff
@@ -339,7 +327,14 @@ fi
 #############################################################
 section_header "Ceph related services"
 
-plugin_command "systemctl status -l 'ceph*'"
+if [ -x /usr/sbin/cephadm ]; then
+    cephadm ls | jq -r '.[] | "\(.fsid) \(.name)"' |
+    while read fsid name ; do
+        plugin_command "systemctl status -l 'ceph-$fsid@$name.service'"
```

Comment:
looks odd to see `$fsid` enclosed within `'` chars (as in `status -l '$fsid'`). (Still, looks like this should work)

Reply:
Fixed.